mrp_production_operation_injection/wizard/mrp_workorder_injector.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class MrpWorkorderInjector(models.TransientModel):
+
+    _name = "mrp.workorder.injector"
+    _description = "Inject operation from BOM into workorders"
+
+    production_id = fields.Many2one("mrp.production", required=True)
+    bom_id = fields.Many2one("mrp.bom", related="production_id.bom_id")
+    allowed_bom_operation_ids = fields.Many2many(
+        "mrp.routing.workcenter", compute="_compute_allowed_bom_operation_ids"
+    )
+    production_workorder_ids = fields.Many2many(
+        "mrp.workorder", compute="_compute_production_workorder_ids"
+    )
+    operation_id = fields.Many2one(
+        "mrp.routing.workcenter", "New operation", required=True
+    )
```

Comment:
isn't it possible to set a domain here, instead of computing the allowed operations?

Reply:
As the domain must be dynamic (according to the `mrp.production` we're injecting an operation on), we need to compute what's allowed anyway.